.pre-commit-config.yaml

```diff
@@ -22,9 +22,11 @@ repos:
       - id: mypy
         additional_dependencies: [types-setuptools]
         files: ^snap7
+        args: [--strict]
```

Comment:
doesn't mypy pick up settings from pyproject.toml?

https://github.com/gijzelaerr/python-snap7/blob/8612423c5729bd64ba4201703479ecd4bfbede61/pyproject.toml#L60

if so we can set strict everyone in one go.

Reply:
fixed